apps/server/src/modules/tldraw/repo/tldraw-board.repo.spec.ts

```diff
@@ -75,7 +56,9 @@ describe('TldrawBoardRepo', () => {
 				const storeGetYDocSpy = jest
 					.spyOn(repo.mdb, 'getYDoc')
 					.mockImplementation(() => Promise.resolve(new WsSharedDocDo('TEST', service)));
-				const storeUpdateSpy = jest.spyOn(repo.mdb, 'storeUpdate').mockImplementation(() => Promise.resolve(1));
+				const storeUpdateSpy = jest
```

Comment:
For promisses please always use .ResolveValue, or RejectValue instead of mock implementations. 
One more please also use on every place the Once too. ResolveValueOnce, RejectValueOnce ..if you need that it is called more then one time in a test. You can add this multiple times as chain. 

Please do this for all tests.
Please also use mock with @golevelup/ts-jest if possible.

Reply:
Changed to mockResolvedValueOnce and mockRejectedValue once where I could